super-ddf.c

```diff
@@ -2297,8 +2297,7 @@ static void getinfo_super_ddf(struct supertype *st, struct mdinfo *info, char *m
 			       be32_to_cpu(ddf->phys->entries[e].refnum) == 0xffffffff)
 				e++;
 			if (i < info->array.raid_disks && e < max &&
-			    !(be16_to_cpu(ddf->phys->entries[e].state)
-			      & DDF_Failed))
+			    !(be16_to_cpu(ddf->phys->entries[e].state) & DDF_Failed))
```

Comment:
Do we need _Missing here?  I tried to follow the threads here, and it looks like this `map` thing only comes into play with some sort of allocated character array?!  Almost all of the callers pass NULL in here, so the whole `map` section isn't executed.

Reply:
👍